bciers/apps/reporting/src/app/components/signOff/SignOffPage.tsx

```diff
@@ -29,11 +30,13 @@ const taskListElements: TaskListElement[] = [
   },
 ];
 
-export default function SignOffPage() {
+const SignOffPage: React.FC<HasReportVersion> = ({ version_id }) => {
```

Comment:
@gdalcengio  
For your consideration:
IMO, for Next.js apps, if your component is a page component, the preferred pattern is a function declarations with export default- `export default function SignOffPage({ version_id }: HasReportVersion) {...`  
This pattern aligns with the framework's dynamic routing and file-based conventions, and since we don't typically need to handle children for page components React.FC adds little value.